R/convert_SST.R

```diff
@@ -0,0 +1,65 @@
+#' Convert SST Data
+#'
+#' @description This function converts SST  data from a given file into a structured format. It extracts information such as start year, end year, duration, longitude, latitude, and SST values from the data file. The function then creates a data frame with the extracted information and returns it along with the latitude-longitude data frame.
+#'
+#' @param datafile A data file containing SST data.
+#' @param data_from The row index from where the SST data starts in the data file. Default is 5.
+#'
+#' @return A list containing the converted data frame and the latitude-longitude data frame.
+#' @export
+#'
+#' @examples
+#' # Example usage:
+#' # Assuming the datafile is "sst_data.csv" and the SST data starts from row 5
+#' converted_data <- convert_SST("sst_data.csv", data_from = 5)
```

Comment:
When the package compiles, it runs all the code. That means, it will try to run this uncommented bit of

`converted_data <- convert_SST("sst_data.csv", data_from = 5)`

However, `sst_data.csv` doesn't actually exist, so we will get an error and have an issue.

The solution is to comment out the example here (just like you have commented out the text above it)